src/utilities/focus.ts

```diff
@@ -1,5 +1,39 @@
+import {
+  findFirstFocusableNode,
+  FOCUSABLE_SELECTOR,
+} from '@shopify/javascript-utilities/focus';
+
 export function handleMouseUpByBlurring({
   currentTarget,
 }: React.MouseEvent<HTMLAnchorElement | HTMLButtonElement>) {
   currentTarget.blur();
 }
+
+export function nextFocusableNode(
+  node: HTMLElement,
+): HTMLElement | Element | null {
+  let nextElement = node.nextElementSibling;
+
+  while (nextElement) {
+    if (nextElement.matches(FOCUSABLE_SELECTOR)) return nextElement;
+
+    if (nextElement instanceof HTMLElement) {
```

Comment:
findFirstFocusableNode require `HTMLElement` rather than `Element`. Although `nextElementSibling` won't return text nodes so I'm not sure we'll ever get an `Element`